qdyn/output.f90

```diff
@@ -752,7 +830,9 @@ subroutine ox_write(pb)
 
   ! Should this proc write ox, ox_dyn, or QSB output?
   write_ox = (mod(pb%it, pb%ox%ntout) == 0 .or. last_call)
-  write_ox_last = last_call
+  !write_ox_last = last_call
+  ! CRP: trying to debug empty write_ox_last
+  write_ox_last = (mod(pb%it, pb%ox%ntout) == 0 .or. last_call)
```

Comment:
I'm guessing that this should not make it into production?...

Reply:
it's (re)writing the last snapshot every ntout times to handle the case that the model is interrupted before the prescribed end of the model (last_call), right?

Reply:
Ah, that's indeed a good idea. Let's keep it that way